vsts/pipelines/PlatformBinaries/dotnetcore.yml

```diff
@@ -0,0 +1,53 @@
+variables:
+  - group: Oryx
+
+stages:
+- stage: Build DotNetCore Platform Binaries
+  jobs:
+  - job: DotNetCore
+    timeoutInMinutes: 250
+    pool:
+      name: OryxLinux
+    steps:
+    - template: templates/_platformBinariesTemplate.yml
+      parameters:
+        platformName: 'dotnet'
+
+- stage: Release
```

Comment:
Let's move this to a separate template since it is a common thing across all platforms here.